libbeat/monitoring/opts.go

```diff
@@ -0,0 +1,46 @@
+package monitoring
+
+// Option type for passing additional options to NewRegistry.
+type Option func(options) options
+
+type options struct {
+	publishExpvar bool
+}
+
+var defaultOptions = options{
+	publishExpvar: false,
+}
+
+// PublishExpvar enables publishing all registered variables via expvar interface.
+// Note: expvar does not allow removal of any stats.
+func PublishExpvar(o options) options {
```

Comment:
yes, it's exactly what I want todo. This `PublishExpvar` is an option one can pass to `monitoring.Registry`. If this is enabled, the metrics created via `monitoring` will be made available in the `expvar` interface as well. To make a custom metric available to `expvar` one has to call `expvar.Publish`, that's why it's named `PublishExpvar`. I'm considering to remove `PublishExpvar` though.